runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -397,14 +397,15 @@ jvmtiGetFrameLocation(jvmtiEnv* env,
 	J9VMThread * currentThread;
 	jmethodID rv_method = NULL;
 	jlocation rv_location = 0;
+	J9InternalVMFunctions* vmFuncs = vm->internalVMFunctions;
```

Comment:
Formatting + initialize local variables. Even though it is old code, we should fix code surrounding our changes to abide by the latest coding standards.
```suggestion
	jvmtiError rc = JVMTI_ERROR_NONE;
	J9VMThread *currentThread = NULL;
	jmethodID rv_method = NULL;
	jlocation rv_location = 0;
	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
```